oam/install_scripts/mcs-workernode.service.in

```diff
@@ -10,7 +10,7 @@ Group=mysql
 LimitNOFILE=65536
 LimitNPROC=65536
 
-ExecStart=@ENGINE_BINDIR@/workernode DBRM_Worker1 fg
+ExecStart=@ENGINE_BINDIR@/workernode DBRM_Worker%i fg
```

Comment:
using mcs-workernode as a template service, we can dynamically call DBRM_WorkerN